vulcan/data/_grade.py

```diff
@@ -48,10 +48,10 @@ class GradeColumn(Serializable):
     period_id: int = IntegerField(key="PeriodId")
     name: str = StringField(key="Name")
     code: str = StringField(key="Code")
-    group: str = StringField(key="Group")
     number: int = IntegerField(key="Number")
     weight: float = FloatField(key="Weight")
     subject: Subject = ChildField(Subject, key="Subject")
+    group: str = StringField(key="Group", required=False)
```

Comment:
I'd suggest to put it above `subject` to make all object parameters last.

Reply:
This is not possible because the not required parameters must be at the end.

Reply:
Oh yes, I didn't notice that.